recipes/h5r/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = 'h5r' %}
+{% set version = 'v1.4.7' %}
+
+package:
+  name: r-{{ name }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://cran.r-project.org/src/contrib/Archive/h5r/h5r_1.4.7.tar.gz
```

Comment:
Here you must use the version variable otherwise the bot may be confused when trying to update this package.